common.sh

```diff
@@ -61,6 +60,13 @@ build_libfuzzer() {
   mv libFuzzer.a $LIB_FUZZING_ENGINE
 }
 
+# Uses the capability for "fsanitize=fuzzer" in the current clang
+build_fsanitize_fuzzer() {
+  CXXFLAGS="-O2 -fno-omit-frame-pointer -g -fsanitize=address,fuzzer"
```

Comment:
Probably what makes the most sense is to have these default flags in addition to user defined flags; if they wish to remove default flags, for some reason, that would have to be after `build_fuzzer` in a `build.sh`.